equinox/nn/_batch_norm.py

```diff
@@ -40,25 +41,69 @@ class BatchNorm(StatefulLayer, strict=True):
     statistics updated. During inference then just the running statistics are used.
     Whether the model is in training or inference mode should be toggled using
     [`equinox.nn.inference_mode`][].
+
+    With `mode = "batch"` during training the batch mean and variance are used
+    for normalization.  For inference the exponential running mean and ubiased
+    variance are used for normalization. This is in line with out other JAX
```

Comment:
Typo.

I'd probably also reference PyTorch instead, which is actually our usual point of reference for Equinox.